app/1.2.x/admin-api.md

```diff
@@ -0,0 +1,2731 @@
+keep the configuration consistent across all nodes.
+
+- `8001` is the default port on which the Admin API listens.
+- `8444` is the default port for HTTPS traffic to the Admin API.
+
+This API is designed for internal use and provides full control over Kong, so
+care should be taken when setting up Kong environments to avoid undue public
+exposure of this API. See [this document][secure-admin-api] for a discussion
+of methods to secure the Admin API.
+
+## Supported Content Types
+
+The Admin API accepts 2 content types on every endpoint:
+
+- **application/x-www-form-urlencoded**
+
+Simple enough for basic request bodies, you will probably use it most of the time.
+Note that when sending nested values, Kong expects nested objects to be referenced
+with dotted keys. Example:
+
+```
+config.limit=10&config.period=seconds
+```
+
+- **application/json**
+
+Handy for complex bodies (ex: complex plugin configuration), in that case simply send
+a JSON representation of the data you want to send. Example:
+
+```json
+{
+    "config": {
+        "limit": 10,
+        "period": "seconds"
+    }
+}
+```
+
+---
+
+## Information Routes
+
+
+
+### Retrieve Node Information
+
+Retrieve generic details about a node.
+
+<div class="endpoint get">/</div>
+
+*Response*
+
+```
+HTTP 200 OK
+```
+
+```json
+{
+    "hostname": "",
+    "node_id": "6a72192c-a3a1-4c8d-95c6-efabae9fb969",
+    "lua_version": "LuaJIT 2.1.0-beta3",
+    "plugins": {
+        "available_on_server": [
+            ...
+        ],
+        "enabled_in_cluster": [
+            ...
+        ]
+    },
+    "configuration" : {
+        ...
+    },
+    "tagline": "Welcome to Kong",
+    "version": "0.14.0"
+}
+```
+
+* `node_id`: A UUID representing the running Kong node. This UUID
+  is randomly generated when Kong starts, so the node will have a
+  different `node_id` each time it is restarted.
+* `available_on_server`: Names of plugins that are installed on the node.
+* `enabled_in_cluster`: Names of plugins that are enabled/configured.
+  That is, the plugins configurations currently in the datastore shared
+  by all Kong nodes.
+
+
+---
+
+### Retrieve Node Status
+
+Retrieve usage information about a node, with some basic information
+about the connections being processed by the underlying nginx process,
+the status of the database connection, and node's memory usage.
+
+If you want to monitor the Kong process, since Kong is built on top
+of nginx, every existing nginx monitoring tool or agent can be used.
+
+
+<div class="endpoint get">/status</div>
+
+*Response*
+
+```
+HTTP 200 OK
+```
+
+```json
+{
+    "memory": {
+		"workers_lua_vms": [{
+			"http_allocated_gc": "0.02 MiB",
+			"pid": 18477
+		}, {
+			"http_allocated_gc": "0.02 MiB",
+			"pid": 18478
+		}],
+		"lua_shared_dicts": {
+			"kong": {
+				"allocated_slabs": "0.04 MiB",
+				"capacity": "5.00 MiB"
+			},
+			"kong_db_cache": {
+				"allocated_slabs": "0.80 MiB",
+				"capacity": "128.00 MiB"
```

Comment:
We need to mention the arguments accepted by this endpoint. Here is the description from the commit (https://github.com/Kong/kong/commit/389c8dd2aeabd90a23c0a7812a43fbb1b7fd9936):

* `unit`: can be one of `b/B`, `k/K`, `m/M`, `g/G`, which will return
    results in bytes, mebibytes, kibibytes, mebibytes, or gibibytes,
    respectively. When "bytes" are requested, the memory values
    in the response will have a number type instead of string.
   Defaults to `k`.
* `scale`: the number of digits to the right of the decimal points when
   values are given in human-readable memory strings (`unit` other than "bytes").
   Defaults to `2`.